Mathlib/SetTheory/Cardinal/Basic.lean

```diff
@@ -1910,8 +1916,9 @@ theorem mk_iUnion_le {α ι : Type u} (f : ι → Set α) : #(⋃ i, f i) ≤ #
 
 theorem mk_iUnion_le_lift {α : Type u} {ι : Type v} (f : ι → Set α) :
     lift.{v} #(⋃ i, f i) ≤ lift.{u} #ι * ⨆ i, lift.{v} #(f i) := by
-  refine mk_iUnion_le_sum_mk_lift.trans <| Eq.trans_le ?_ (sum_le_iSup_lift _)
-  rw [← lift_sum, lift_id'.{_,u}]
+  apply mk_iUnion_le_sum_mk_lift.trans
+  convert sum_le_iSup_lift fun i ↦ lift.{v} #(f i)
+  rw [← lift_sum, lift_id']
```

Comment:
I still prefer the old proof here, fwiw, refine and Eq.trans_le are usually nicer than convert. But I think a calc proof might be even better than both?

Reply:
I'll keep the original proof then.